homeassistant/components/lutron_caseta/cover.py

```diff
@@ -78,3 +68,79 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
         if ATTR_POSITION in kwargs:
             position = kwargs[ATTR_POSITION]
             await self._smartbridge.set_value(self.device_id, position)
+
+
+class LutronCasetaTiltOnlyBlind(LutronCasetaCoverBase):
+    """Representation of a Lutron tilt only blind."""
+
+    _attr_supported_features = (
+        CoverEntityFeature.OPEN_TILT
+        | CoverEntityFeature.CLOSE_TILT
+        | CoverEntityFeature.STOP_TILT
+        | CoverEntityFeature.SET_TILT_POSITION
+        | CoverEntityFeature.OPEN_TILT
+    )
+    _attr_device_class = CoverDeviceClass.BLIND
+
+    @property
+    def is_closed(self) -> bool:
+        """Return if the cover is closed."""
+        return self._device["tilt"] == 0 or self._device["tilt"] == 100
+
+    @property
+    def current_cover_tilt_position(self) -> int:
+        """Return the current position of cover."""
+        return self._device["tilt"]
+
+    async def async_close_cover_tilt(self, **kwargs: Any) -> None:
+        """Close the cover."""
+        await self._smartbridge.set_tilt(self.device_id, 0)
+        await self.async_update()
+        self.async_write_ha_state()
+
+    async def async_open_cover_tilt(self, **kwargs: Any) -> None:
+        """Open the cover."""
+        await self._smartbridge.set_tilt(self.device_id, 50)
+        await self.async_update()
+        self.async_write_ha_state()
+
+    async def async_set_cover_tilt_position(self, **kwargs: Any) -> None:
+        """Move the shade to a specific position."""
+        if ATTR_TILT_POSITION in kwargs:
```

Comment:
`ATTR_TILT_POSITION` is required so this can't be missing. Please remove the guard

https://github.com/home-assistant/core/blob/4da701a8e9ee880fbbfbaec24c237e1e6146a509/homeassistant/components/cover/__init__.py#L224